neutrons/data_workflow
# REF_M: Add the capability to autoreduce two samples from the same run

References [EWM 5797](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20%28Change%20Management%29#action=com.ibm.team.workitem.viewWorkItem&id=5797)

# Short description of the changes:
Update the reduction options form for instrument `REF_M` with entries for up to three samples.

# Check list for the pull request
- [x] I have updated tests for my changes

# Check list for the reviewer

# Manual test for the reviewer
Run unit test `test_ref_m`  but first [remove comment to print the HTML form](https://github.com/neutrons/data_workflow/blob/refm_multi_samples/src/webmon_app/reporting/tests/test_reduction/test_views.py#L201). After that point your browser to `file:///tmp/junk.html` to see the form with entries for three samples.
